tests/cli/test_cmds_local_run.py

```diff
@@ -344,7 +349,8 @@ def test_configure_and_run_pulls_image_when_asked(
         command=['bash'],
         healthcheck=args.healthcheck,
         healthcheck_only=args.healthcheck_only,
-        instance_config=mock_get_instance_config.return_value
+        instance_config=mock_get_instance_config.return_value,
+        soa_dir=args.yelpsoa_config_root
```

Comment:
go ahead and add a trailing comma so we don't get a 3-line diff the next time someone adds an arg.